go.mod

```diff
@@ -5,22 +5,22 @@ go 1.13
 replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
 	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20190404221404-ee5226d43009
-	k8s.io/client-go => k8s.io/client-go v0.17.2
+	k8s.io/client-go => k8s.io/client-go v0.18.0
 )
 
 require (
 	github.com/coreos/prometheus-operator v0.36.0
 	github.com/knative/pkg v0.0.0-20190817231834-12ee58e32cc8
 	github.com/pkg/errors v0.8.1
-	github.com/rancher/norman v0.0.0-20200211155126-fc45a55d4dfd
-	github.com/rancher/wrangler v0.5.0
-	github.com/rancher/wrangler-api v0.5.0
+	github.com/rancher/norman v0.0.0-20200326201949-eb806263e8ad
```

Comment:
Need to update this after merging https://github.com/rancher/norman/pull/364 with the commit from the release/v2.4 branch